collector/main.go

```diff
@@ -26,45 +24,64 @@ import (
 	"github.com/open-telemetry/opentelemetry-lambda/collector/extension"
 	"github.com/open-telemetry/opentelemetry-lambda/collector/lambdacomponents"
 	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 )
 
 var (
 	extensionName   = filepath.Base(os.Args[0]) // extension name has to match the filename
 	extensionClient = extension.NewClient(os.Getenv("AWS_LAMBDA_RUNTIME_API"))
-	logger          = zap.NewExample()
+	logger          *zap.SugaredLogger
 )
 
 func main() {
+	configureLogger()
+	defer logger.Sync()
 
-	logger.Debug("Launching OpenTelemetry Lambda extension", zap.String("version", Version))
+	logger.Debugw("Launching OpenTelemetry Lambda extension", "version", Version)
 
 	factories, _ := lambdacomponents.Components()
 	collector := NewCollector(factories)
 	ctx, cancel := context.WithCancel(context.Background())
 
 	if err := collector.Start(ctx); err != nil {
-		log.Fatalf("Failed to start the extension: %v", err)
+		logger.Fatalf("Failed to start the extension: %v", err)
 	}
 
 	sigs := make(chan os.Signal, 1)
 	signal.Notify(sigs, syscall.SIGTERM, syscall.SIGINT)
 	go func() {
 		s := <-sigs
 		cancel()
-		logger.Debug(fmt.Sprintf("Received", s))
+		logger.Debugf("Received: %s", s.String())
 		logger.Debug("Exiting")
 	}()
 
 	res, err := extensionClient.Register(ctx, extensionName)
 	if err != nil {
-		log.Fatalf("Cannot register extension: %v", err)
+		logger.Fatalf("Cannot register extension: %v", err)
 	}
 
-	logger.Debug("Register ", zap.String("response :", prettyPrint(res)))
+	logger.Debugw("Register", "response", res)
 	// Will block until shutdown event is received or cancelled via the context.
 	processEvents(ctx, collector)
 }
 
+func configureLogger() {
+	atom := zap.NewAtomicLevel()
+
+	level, err := zapcore.ParseLevel(os.Getenv("OPENTELEMETRY_EXTENSION_LOG_LEVEL"))
```

Comment:
Can you add in PR description to indicate what keyword should be assigned to 
`OPENTELEMETRY_EXTENSION_LOG_LEVEL` in order to set log level to debug.

Reply:
From what I see, it accepts only a specific keywoard for loglevel and if there is none or wrong it sets it as debug.
Look at what zapcore.ParseLevel does.

Reply:
Some clarification to my previous comment: if os.getEnv returns an empty string zapcore.ParseLevel will parse it as info.
`func (l *Level) unmarshalText(text []byte) bool {
	switch string(text) {
	case "debug", "DEBUG":
		*l = DebugLevel
	case "info", "INFO", "": // make the zero value useful
		*l = InfoLevel
`
Thats why you dont see any logs when there's no env var provided.

Reply:
ah, I didn't realize that `""` check existed, good to know.